app/services/schemas.py

```diff
@@ -1,7 +1,7 @@
 from pydantic import BaseModel
 from typing import Optional, List, Any
 from enum import Enum
-from app.services.tool_registry import BaseTool
+from .tool_registry import BaseTool
```

Comment:
All imports should use absolute paths instead of relative ones, as relative paths won't work in production. Please change all import paths to start with `app`.